app/experimenter/experiments/models.py

```diff
@@ -614,7 +632,28 @@ def completed_objectives(self):
 
     @property
     def completed_results(self):
-        return self.results_url or self.results_initial or self.results_lessons_learned
+        results_fields = (
+            "results_url",
+            "results_initial",
+            "results_lessons_learned",
+            "results_fail_to_launch",
+            "results_recipe_errors",
+            "results_restarts",
+            "results_low_enrollment",
+            "results_early_end",
+            "results_no_usable_data",
+            "results_failures_notes",
+            "results_changes_to_firefox",
+            "results_data_for_hypothesis",
+            "results_confidence",
+            "results_measure_impact",
+            "results_impact_notes",
+        )
+        for field in results_fields:
+            if getattr(self, field):
+                return True
+
```

Comment:
would something along the lines of this work and be cleaner? (idk if i'm really sure this would work)
```python
results_fields = (self. results_url, self.results_intial, <and the rest of the> ...)
return any(results_fields)
```

Reply:
Yeah I think following the pattern of the risks we could just do something like

        any([getattr(field, self) for field in results_fields])

Reply:
thanks all.